templates/explore/repo_list.tmpl

```diff
@@ -11,6 +11,10 @@
 				</a>
 				{{if .IsPrivate}}
 					<span class="middle text gold">{{svg "octicon-lock" 16}}</span>
+				{{else if .Owner}}
```

Comment:
I think this will cause all Forks and Mirrors to lose their icons...

Reply:
Good catch, should be fixed now.